sdk/compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -1045,7 +1045,9 @@ multiPackageBuildEffect relativize mPkgConfig multiPackageConfig projectOpts opt
     IDE.shutdown
     $ \ideState -> runActionSync ideState
       $ case mRootPkgData of
-          Nothing -> void $ uses_ BuildMulti $ toNormalizedFilePath' <$> mpPackagePaths multiPackageConfig
+          -- `uses_` tries to run in "parallel", but ends up just running backwards for multi-build.
+          -- We use `reverse` here to keep the build order consistent with the multi-package.yaml
+          Nothing -> void $ uses_ BuildMulti $ reverse $ toNormalizedFilePath' <$> mpPackagePaths multiPackageConfig
```

Comment:
Naive question.
Why do we need the `reverse` here ?

Reply:
Its explained in the comment above :)

Reply:
Thanks